### PR TITLE
Annotation of PXD005507

### PR DESCRIPTION
**Pull request by Ohwey**

### CODE REVIEW

File: annotated-projects/PXD005507/sdrf.tsv [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+sample1	homo sapiens	"NT=Liver;AC=NCIT:C12392"	epithelial cell	"NT=hepatocellular carcinoma;AC=DOID:684"	"NT=HepG2;AC=EFO:0001187"	OEII15637	OEII15637.raw	ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2017/05/PXD005507/OEII15637.raw	1	label free sample	"NT=LTQ Orbitrap Elite;AC=MS:1001910"	linear ion trap		35% NCE	collision-induced dissociation	"NT=Trypsin; AC=MS:1001251; CS=(??[KR])(?!P)"	"NT=Oxidation; MT=Variable; AC=Unimod:35; TA=M"	"NT=Carbamidomethyl; MT=Fixed; AC=Unimod:4; TA=C"	5 ppm	0.4 Da	1	Cu complex	cell
+sample1	homo sapiens	"NT=Liver;AC=NCIT:C12392"	epithelial cell	"NT=hepatocellular carcinoma;AC=DOID:684"	"NT=HepG2;AC=EFO:0001187"	OEII15635	OEII15635.raw	ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2017/05/PXD005507/OEII15635.raw	1	label free sample	"NT=LTQ Orbitrap Elite;AC=MS:1001910"	linear ion trap		35% NCE	collision-induced dissociation	"NT=Trypsin; AC=MS:1001251; CS=(??[KR])(?!P)"	"NT=Oxidation; MT=Variable; AC=Unimod:35; TA=M"	"NT=Carbamidomethyl; MT=Fixed; AC=Unimod:4; TA=C"	5 ppm	0.4 Da	1	Cu complex	cell
```
[the remark this answers is not in the document]
Comment (Ohwey) [reply annotated-projects/PXD005507/sdrf.tsv]: My bugfix commit solves this issue. Each different disease has now it's own unique source name

**Review by ypriverol — CHANGES_REQUESTED** [review]
Dear @ohwey I have difficulties to understand the SDRF you have same sample 1 associated with two different files and not fractionization. Are this technical replicates?

File: annotated-projects/PXD005507/sdrf.tsv (hunk 1 above)
[the remark this answers is not in the document]
Comment (ypriverol) [reply annotated-projects/PXD005507/sdrf.tsv]: Dear @ohwey I have difficulties to understand the SDRF you have same sample 1 associated with two different files and not fractionization. Are these technical replicates?

**Comment by Ohwey** [comment]
Dear @ypriverol in this experiment are 3 treatment groups used. Each treatment group uses 6 experimental/biological replicates. There are no technical or fractional replicates in this experiment.

**Pull request merged**